### ISSUE TITLE
Value ranges for functions

### ISSUE THREAD

**Issue by dawhitefish** [mrjbq7/ta-lib#126]
Is there a way to find out the min and max values for each of the functions?

**Comment by mrjbq7** [comment]
Hmm, I assume you mean programmatically, and I'm not sure if that's available.  Would have to look through some of the Abstract API features in the underlying C library.

**Comment by dawhitefish** [comment]
Yes, I just wanted to see if it was possible programmatically. If its not I guess I will just have to find the values myself. Thank you for the quick response

**Comment by mrjbq7** [comment]
You might look through the ``output_flags`` of the Abstract API:

```python
>>> import talib.abstract
>>> talib.abstract.MOM.info
>>> talib.abstract.MOM.output_flags
```

The ``output_flags`` is an enum with these possibilities:

https://github.com/mrjbq7/ta-lib/blob/master/talib/abstract.pyx#L496
